CHANGELOG.md

```diff
@@ -8,6 +8,21 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
```

Comment:
If we wouldn't add empty headings, we wouldn't need to remove them before the next release :wink: